src/redisraft.c

```diff
@@ -621,6 +663,10 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
         return;
     }
 
+    if (handleUnwatch(rr, ctx, cmds)) {
```

Comment:
I'm scared to ask this question but hope ACL does not apply to unwatch :) 
dry-run is a bit down below, we skip ACL check currently, maybe we can move this down a bit.

Reply:
it can.  so, we should.  moving.